projects/demo-playwright/tests/kit/input/input.spec.ts

```diff
@@ -0,0 +1,132 @@
+
+    test('correctly aligns single custom content (as large icon)', async ({page}) => {
+        await tuiGoto(
+            page,
+            '/components/input/API?tuiTextfieldCustomContent=tuiIconCalendarLarge',
+        );
+        await expect(new TuiDocumentationPagePO(page).apiPageExample).toHaveScreenshot(
+            '02-input.png',
+        );
+    });
+
+    test("custom content (as large icon) + cleaner + hint don't overlapping each others", async ({
+        page,
+    }) => {
+        await tuiGoto(
+            page,
+            '/components/input/API?tuiTextfieldCleaner=true&tuiTextfieldCustomContent=tuiIconSearchLarge&tuiHintContent=Some%20content',
+        );
+
+        await expect(new TuiDocumentationPagePO(page).apiPageExample).toHaveScreenshot(
+            '03-input.png',
+        );
+    });
+
+    test("custom content (as normal-size icon) + cleaner + hint don't overlapping each others", async ({
+        page,
+    }) => {
+        await tuiGoto(
+            page,
+            'components/input/API?tuiTextfieldCleaner=true&tuiTextfieldCustomContent=tuiIconVisaMono&tuiHintContent=Some%20content',
+        );
+
+        await expect(new TuiDocumentationPagePO(page).apiPageExample).toHaveScreenshot(
+            '04-input.png',
+        );
+    });
+
+    test('input overflow due to placeholder', async ({page}) => {
+        await tuiGoto(
+            page,
+            'components/input/API?tuiTextfieldIconLeft=tuiIconSearch&pseudoFocus=true&placeholder=Lorem%20ipsum%20dolor%20sit%20amet,%20consectetur%20adipiscing%20elit,%20sed%20do%20eiusmod%20tempor%20incididunt%20ut%20labore',
+        );
+
+        const example = new TuiDocumentationPagePO(page).apiPageExample;
+        const input = example.locator('.t-input-wrapper').locator('input[tuiTextfield]');
+
+        await input.focus();
+        await expect(new TuiDocumentationPagePO(page).apiPageExample).toHaveScreenshot(
+            '05-input.png',
+        );
+        await input.clear();
+        await input.blur();
+        await expect(example).toHaveScreenshot('06-input.png');
+    });
+
+    test('character descenders overflow the line', async ({page}) => {
+        await tuiGoto(
+            page,
+            '/components/input/API?&pseudoFocus=true&placeholder=big, placeholder, qwerty, jackson, yellow and more',
+        );
+
+        const example = new TuiDocumentationPagePO(page).apiPageExample;
+        const input = example.locator('.t-input-wrapper').locator('input[tuiTextfield]');
+
+        await input.clear();
+        await expect(example).toHaveScreenshot('07-input.png');
+        await input.focus();
+        await page.keyboard.type(
+            "It has been the industry's standard dummy text ever since the 1500s",
+        );
+        await page.keyboard.press('Enter');
+        await input.blur();
+        await expect(example).toHaveScreenshot('08-input.png');
+    });
+
+    test('can be horizontally scrolled', async ({page}) => {
```

Comment:
This test does not have any action with scroll (see previous cypress test).
This test does not complete its main goal (according to title).

Reply:
fixed